packages/asset-kit-block/src/AssetKitBlock.tsx

```diff
@@ -39,24 +81,35 @@ export const AssetKitBlock = ({ appBridge }: BlockProps): ReactElement => {
                 <div className="tw-flex-none">
                     <button
                         data-test-id="asset-kit-block-download-button"
-                        disabled={isDownloadingAssets || isUploadingAssets || currentAssets.length <= 0}
-                        onClick={() =>
-                            generateBulkDownload(appBridge.getProjectId(), currentAssets, setIsDownloadingAssets)
+                        disabled={
+                            [BulkDownloadState.Error, BulkDownloadState.Pending, BulkDownloadState.Started].includes(
+                                status
+                            ) || currentAssets.length === 0
                         }
+                        onClick={() => startDownload()}
                         style={designTokens.buttonPrimary}
                     >
                         Download package
+                        <span
+                            data-test-id="asset-kit-block-screen-reader"
+                            ref={screenReaderRef}
+                            role="status"
+                            className="tw-absolute -tw-left-[10000px] tw-top-auto tw-w-1 tw-h-1 tw-overflow-hidden"
```

Comment:
Not sure if tw-overflow-hidden has any effect here, since it would control the overflow mechanics of its content, but it has no content. I think what you maybe meant to do is add overflow-hidden to one of the parent elements?